discover/portal/articles/130-forms/04-form-rules/05-autofill-rules.markdown

```diff
@@ -1,11 +1,11 @@
 # Action: Autofill [](id=action-autofill)
 
 Autofill rules let you change the selection options of another field based
-on the value entered into a related field. A [data provider's]((/discover/portal/-/knowledge_base/7-1/data-providers)) output is used to
+on the value entered into a related field. A [data provider's]((/discover/portal/-/knowledge*base/7-1/data-providers)) output is used to
```

Comment:
The parentheses are duplicated, is it on purpose?

Reply:
Note on purpose. Thanks.